### PR TITLE
[CBRD-22371] Insert multiple records at a time into heap

### PR DESCRIPTION
**Pull request by lucianlazar2211**
http://jira.cubrid.org/browse/CBRD-22371

This PR integrates the `locator_multi_insert_force` function into the `loaddb` module. It collects an entire batch of records and issues the insert of multiple records by forming a chain of heap pages. Then it links this chain to the heap file and commits.

### CODE REVIEW

File: src/transaction/locator_sr.c [hunk 1]
```diff
@@ -13766,19 +13768,21 @@ locator_multi_insert_force (THREAD_ENTRY * thread_p, HFID * hfid, OID * class_oi
 	        }
 
 	      for (size_t j = 0; j < recdes_array.size (); j++)
-	        {
+		{
```
Comment (eseokoh) [inline src/transaction/locator_sr.c]: I think we need stats for `locator_multi_insert_force`:
stats for multi-insert
avg num of objects inserted to a page
...

You may separate the task from this.

File: CMakeLists.txt [hunk 2]
```diff
@@ -365,7 +365,7 @@ else(BISON_FOUND)
       " and retry running cmake."
     )
   endif()
-endif(NOT BISON_FOUND)
+endif(BISON_FOUND)
```
Comment (ribram) [inline CMakeLists.txt]: is this change intended?

Comment (valerinistor) [reply CMakeLists.txt]: Yes, it is intended. cmake throws a warning since a logical block closes with mis-matching arguments

File: src/transaction/locator_sr.c [hunk 3]
```diff
@@ -13789,25 +13795,30 @@ locator_multi_insert_force (THREAD_ENTRY * thread_p, HFID * hfid, OID * class_oi
...
+
+	      assert (!pgbuf_is_page_fixed_by_thread (thread_p, &new_page_vpid));
 	    }
 
 	  // Add this record to the recdes array and increase the accumulated size.
-	  recdes_array.push_back (recdes[i]);
-	  accumulated_records_size += recdes[i].length;
-        }
+	  recdes_array.push_back (local_record);
+	  accumulated_records_size += local_record.length;
```
Comment (alexstanro) [inline src/transaction/locator_sr.c]: Do we need alignment here?
